cmd/spire-agent/cli/run/run.go

```diff
@@ -55,11 +55,12 @@ type Config struct {
 }
 
 type agentConfig struct {
-	DataDir                       string    `hcl:"data_dir"`
-	AdminSocketPath               string    `hcl:"admin_socket_path"`
-	InsecureBootstrap             bool      `hcl:"insecure_bootstrap"`
-	JoinToken                     string    `hcl:"join_token"`
-	LogFile                       string    `hcl:"log_file"`
+	DataDir                       string `hcl:"data_dir"`
+	AdminSocketPath               string `hcl:"admin_socket_path"`
+	InsecureBootstrap             bool   `hcl:"insecure_bootstrap"`
+	JoinToken                     string `hcl:"join_token"`
+	LogFile                       string `hcl:"log_file"`
+	LogReopener                   log.ReopenableWriteCloser
```

Comment:
Absolutely. I was concerned about that exact point. My first instinct was to make it unexported, but I didn't see an extant example. I went with grouping near the relevant `LogFile` config. I figured it would elicit some response with guidance.